api/sixtyfps-cpp/include/sixtyfps.h

```diff
@@ -365,6 +364,29 @@ inline LayoutInfo LayoutInfo::merge(const LayoutInfo &other) const
                         std::min(vertical_stretch, other.vertical_stretch) };
 }
 
+/// FIXME! this should be done by cbindgen
+namespace cbindgen_private {
+inline bool operator==(const LayoutInfo &a, const LayoutInfo &b)
```

Comment:
Just a thought, but while this is done manually perhaps it would be worthwhile to add a comment on the Rust side of the data structure declaration to remind anyone to make changes here when adding new fields (removal or rename will cause build breakage, good, but addition will silently "pass").